frontend/src/components/navbar/AuditSidebar.tsx

```
@@ -163,6 +183,100 @@ const AuditSidebar = ({
+                  'flex items-center gap-2 text-sm w-full justify-start px-4 py-2 text-left font-medium rounded-lg transition-colors duration-200',
+                  activeItem === section.name
+                    ? 'bg-gray-800 text-white'
+                    : 'text-gray-400',
+                  'hover:bg-gray-800',
+                )}
+                onClick={() => {
+                  setActiveItem(section.name);
+                }}
+                to={`sections/${section._id}`}
+              >
+                {section.icon && section.icon.startsWith('fa-') ? (
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)